contracts/serverless-v2/GatewayJobs.sol

```diff
@@ -179,7 +180,9 @@ contract GatewayJobs is
 
     function setJobAllowance() external onlyRole(DEFAULT_ADMIN_ROLE) {
         // increasing allowance to be used while relaying jobs
-        USDC_TOKEN.safeIncreaseAllowance(address(JOB_MANAGER), type(uint256).max);
+        bool success = USDC_TOKEN.approve(address(JOB_MANAGER), type(uint256).max);
+        if(!success)
+            revert GatewaysJobsApprovalFailed();
```

Comment:
can you add USDC in the error name? Also, add spendor address and amount as argument, will be helpful in debugging.

Reply:
> can you add USDC in the error name? Also, add spendor address and amount as argument, will be helpful in debugging.

@vg-27 Can I contribute to the project?